node-src/tasks/verify.ts

```diff
@@ -249,7 +249,10 @@ export const verifyBuild = async (ctx: Context, task: Task) => {
 
   if (ctx.build.wasLimited) {
     const { account } = ctx.build.app;
-    if (account.exceededThreshold) {
+    if (!account) {
+      ctx.log.warn('No Account');
+      setExitCode(ctx, exitCodes.INVALID_OPTIONS, true);
```

Comment:
This wouldn't be `INVALID_OPTIONS` but something more severe, and probably not user error. Basically this should never happen so throwing an error would be the proper recourse here.

```suggestion
      throw new Error('Failed to retrieve account information');
```